index.html

```diff
@@ -53,7 +53,7 @@ <h3 class="form-label">Seconds</h3>
         </form>
       </section>
       <section class="past-activity">
-        <h2 class="subhead">Past Activities</h2>
+        <h2 class="subhead top">Past Activities</h2>
```

Comment:
please remove the unnecessary class of "top".

Reply:
removed.